examples/heal/local-forwarder-remote-forwarder/README.md

```diff
@@ -0,0 +1,159 @@
+# Local Forwarder + Remote Forwarder restart
```

Comment:
Consider using 
```
kubectl scale --replicas=0 daemonset/forwarder-vpp -n nsm-system
```
To kill both forwarder instances
and re-create them with
```
kubectl scale --replicas=1 daemonset/forwarder-vpp -n nsm-system
```

This can a bit reduce the complexity of this test.

Reply:
I think we cannot do it so easily, because we use `daemonset`, not `deployment`. And we cannot use `--replicas` in this case.